config/v2/config.go

```diff
@@ -84,6 +89,7 @@ type Common struct {
 	TerraformVersion         *string           `yaml:"terraform_version,omitempty"`
 	Tools                    *Tools            `yaml:"tools,omitempty"`
 	NeedsAWSAccountsVariable *bool             `yaml:"needs_aws_accounts_variable,omitempty"`
+	ExtraTemplates           *[]ExtraTemplate  `yaml:"extra_template,omitempty"`
```

Comment:
Should this be plural?
```suggestion
	ExtraTemplates           *[]ExtraTemplate  `yaml:"extra_templates,omitempty"`
```

Reply:
nice catch